css/main.css

```diff
@@ -56,19 +61,177 @@ textarea {
    Author's custom styles
    ========================================================================== */
 
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.black {
+  color: #000;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.main {
+  display: flex;
+  overflow: hidden;
+  font: normal 16px sans-serif;
+  width: 100%;
+  height: 100%;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  background: #0e171d;
+  color: #fff;
+  width: 300px;
+  height: 100%;
+  float: left;
+}
+
+.logo-motorpool {
+  background-image: url("../img/logo.png");
+  background-repeat: no-repeat;
+  margin-left: 35px;
+  margin-top: 40px;
+  margin-bottom: 46px;
+  min-width: 67px;
+  min-height: 43px;
+}
+
+form.search-form {
```

Comment:
No need to specify the `form` part of this selector — it's actually very slightly slower with an element in it.